src/main/java/com/checkmarx/flow/service/SastScanner.java

```diff
@@ -116,7 +117,7 @@ public boolean isEnabled() {
         if (enabledScanners == null || enabledScanners.isEmpty()) {
             log.info("None of the vulnerability scanners is enabled in the configuration. Using CxSAST scanner by default.");
             result = true;
-        } else if (enabledScanners.contains(CxProperties.CONFIG_PREFIX)) {
+        } else if (StringUtils.containsIgnoreCase(enabledScanners.toString(), CxProperties.CONFIG_PREFIX)) {
```

Comment:
The name of CONFIG_PREFIX constant is not clear (Scanner type?)

Reply:
This is the scanner defined constant in the SDK's properties files. We wanted to re-use the variable instead of defining a new one.